src/polytax/train.py

```diff
@@ -148,10 +148,12 @@ def _build_tasks(self, train_state: "TrainState"):
         if xla_found and self.IS_LOCAL_MASTER:
             xm.rendezvous("download_only_once")
 
+    # Is it needed to have train_state as an argument?
     def _build_train_tasks(self, train_state: "TrainState"):
         self.train_task = get_task(**self.get("dataset/kwargs"))
         self.train_loader = get_dataset(task=self.train_task, seed=self.get("seed"), GLOBAL_RANK=self.GLOBAL_RANK, NUM_SHARDS=self.NUM_SHARDS, device=self.device, **self.get("dataset/kwargs"),)
 
+    # Same question as _build_train_tasks
```

Comment:
no it's not needed, just for consistency. Can be removed if you like.

Reply:
but pleae remove these comments and instead use the github review system or slack for asking questions